controllers/clustercache/cluster_accessor.go

```diff
@@ -264,6 +266,7 @@ func (ca *clusterAccessor) Connect(ctx context.Context) (retErr error) {
 	}
 
 	log.Info("Connected")
+	connectionUp.WithLabelValues(ca.cluster.String()).Set(1)
```

Comment:
Should we move this into an else branch in the defer? Then the metric is handled in a single place